MVP/defaults-build.yaml

```diff
@@ -12,18 +12,20 @@
             script-only-if-succeeded: False
             script-only-if-failed: True
             builders:
-              - shell: |
-                  #!/bin/bash
-                  echo "Jenkins job failed or canceled - canceling any submitted Beaker jobs"
-                  bkr job-cancel ${{BKR_JOBID}}
+                - shell: |
+                    echo "Jenkins job failed or canceled - canceling any submitted Beaker jobs"
+                    bkr job-cancel ${{BKR_JOBID}}
         - archive:
             artifacts: '*.txt, *.xml, *.properties'
             allow-empty: 'true'
         - junit:
             results: 'J:*.xml'
         - email-ext:
             recipients: ${{OWNERSHIP,var="JOB_COOWNERS_EMAILS"}}
-            subject: $DEFAULT_SUBJECT [${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}]
+            subject: $DEFAULT_SUBJECT [${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}] (${{TEST_COUNTS,var="fail"}}/${{TEST_COUNTS}})
+            body: |
+                $DEFAULT_CONTENT
+                Beaker job at https://beaker.engineering.redhat.com/matrix/?job_ids=${{ENV,var="BKR_JOBID_NUM"}}
```

Comment:
Good point - actually i've been thinking about this for a while.
Using jenkins global var would definitely be a decent solution - especially for internal URLs with sensitive RH info.

At this moment, people can try MVP on any jenkins including PJM (since current MVP has little dependency with jenkins env/plugin). I was hesitate to lose this advantage right now when some users haven't finished the transition - also when beaker url is considered less sensitive (like PJM url which approved by infosec) in this case.

But I share your concern here, and maybe it's also a good thing to give folks one more reason of moving to PJM.
WIP.